src/components/DeleteItemButton.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { doc, deleteDoc } from '@firebase/firestore';
+import { db } from '../lib/firebase';
+import Modal from './Modal';
+
+function DeleteItemButton(item) {
```

Comment:
Suggestion: You can destructure the prop to show that you are passing in `userToken` from Item.js. 
```suggestion
function DeleteItemButton({ item, userToken }) {
```

Reply:
Updated, thank you for pointing this out!